jekyll/_cci2_ja/runner-overview.adoc

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
@mfunaki My review isn't applied yet, could you double-check it?
https://circleci.atlassian.net/browse/DNAMEDIA-79

Reply:
Sorry, I've re-created a new PR #4980 where I've applied your modifications.  Thanks!